### PR TITLE
Back to top link added.🔝

### LINKED ISSUE #1755: [GSSoC'23][Documentation Bug]: Back to Top link is missing at the bottom of the ReadMe.
# ISSUE-

Since, **ReadMe's grow in size as more and more contributors join** and in our case **List of Available Projects** section in the **ReadMe** is going to make our ReadMe grow in size in future. Hence, adding **link to go back to the top of the page** is necessary **for better readability.**

**@pranjay-poddar**  Kindly assign me this issue.

----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

# FEATURE-
I will add Back to the top link similar to as shown below.

back-to-top-link

### PR DESCRIPTION
**Pull request by arcVaishali**
# Back to Top Link missing from the Readme #1755 

Back to Top link added🔝

Close #1755 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: please not change this file

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (arcVaishali) [reply .vscode/settings.json:1]: Okay I will check and revert changes

**Review by arcVaishali — COMMENTED** [review]
Reverted the changes made in .vscode/settings.json

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: @arcVaishali why you remove all content from this , remove the file changes of this file from your PR.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (arcVaishali) [reply .vscode/settings.json:1]: Okay, I misunderstood. I have to add this file in gitignore folder. I guess then only, changes made to this file will not be a part of this PR?

Comment (arcVaishali) [reply .vscode/settings.json:1]: I have removed .vscode folder completely

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**